IvanTurchin/src/moto/Biker.java

```diff
@@ -0,0 +1,87 @@
+package moto;
+
+public class Biker {
+
+    private String name;
+    private String phone;
+    private String address;
+    private int money;
+    private boolean tools;
+
+    private Motorcycle motorcycle = null;
+
+    public Biker(String name, String phone, String address, int money) {
+        this.name = name;
+        this.phone = phone;
+        this.address = address;
+        this.money = money;
+    }
+
+    public void buyMotorcycle(String model) {
```

Comment:
This method should be in BikeShop 
Don't create MotoShop for buying moto within Biker class